store/kv/kv.go

```diff
@@ -102,6 +103,22 @@ func measureLow(ctx context.Context, name string, f func() error) {
 		metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets)
 		defer metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets).Start().Stop()
 	}
+	if config.DefaultConfig.DatadogTrace.Enabled {
+		spanOpts := []tracer.StartSpanOption{
+			tracer.ServiceName("tigris.fdb.kv"),
+			tracer.ResourceName(name),
+			tracer.SpanType("fdb_kv"),
+			tracer.Measured(),
+		}
+		parentSpan, exists := tracer.SpanFromContext(ctx)
```

Comment:
move the spanOpts initialization inside the `if exists` check.

Reply:
Good catch, moved.